README.md

```diff
@@ -42,8 +45,8 @@ From the `./web` directory run `npm install -D @next/swc-darwin-arm64`
 1.  Run `npm run dev:emulator`
 
     - Local development uses the [Flow Emulator](https://docs.onflow.org/emulator/) and the [FCL Development Wallet](https://github.com/onflow/fcl-dev-wallet) to simulate the blockchain and an FCL-compatible wallet.
+    - Running this npm script initializes your application by completing multiple important processes found [here](/.ki-scripts/README.md)
 
-2.  In another terminal, from the root of the project run `flow dev-wallet`
 
 Thats it! 🏁
```

Comment:
It probably doesn't make sense to use a numbered list now that there's only 1 step.

I'd recommend starting with a simple instruction and then link out to the scripts README afterwards. This minimizes the amount of text the developer needs to read.

I also find it better to put commands in full code blocks (i.e. three back ticks) because it's easier to copy.

---

## Local development

Run this command to start Kitty Items with the Flow local development suite:

```sh
npm run dev:emulator
```

Note: this script launches the Flow Emulator and FCL Development Wallet using the [initialization code in this repository](/.ki-scripts/README.md).